conf/server.yaml.template

```diff
@@ -1,6 +1,29 @@
 SERVER:
-  # Server hostname
-  # HOSTNAME: replace-with-satellite-hostname
+  # Server hostnames, a list of one or more dns-resolvable hostnames
+  HOSTNAMES:
+    - null
+  #  - replace.with.satellite.hostname
+  #  - replace.with.satellite.hostname
+  VERSION:
+    # The full release version (6.9.2)
+    # RELEASE: populate with satellite version
+    # The snap version currently testing (if applicable)
+    # SNAP:
+    # The source of Satellite packages. Can be one of:
+    # internal, ga, beta
+    SOURCE: "internal"
+  # run-on-one - All xdist runners default to the first satellite
+  # balance - xdist runners will be split between available satellites
+  # on-demand - any xdist runner without a satellite will have a new one provisioned.
```

Comment:
It will indeed delay execution until a satellite is received (if a new one is needed) I can add a line stating this but might be overkill for a setting explanation.

Reply:
execution only for that xdist worker will be halted. I'd agree its too much to describe here in the in-line comments. Readme or docblock for the hook should describe this behavior in more detail.